src/frontend/src/pages/PPQPage/PPQPage.tsx

```diff
@@ -5,7 +5,12 @@ import ppqImg from '../../assets/ppq_homepage.svg';
 
 import { Link } from 'react-router-dom';
 
-function PPQLandingPage() {
+interface IComponentProps {
+  pia: boolean;
+}
+function PPQLandingPage(props: IComponentProps) {
+  // TODO use pia value to decide render pia intake or not
```

Comment:
is it still pending? Looks like it is. Please mark this PR as draft

Reply:
No this is not pending status, as 401 is not finished yet, we can not really check the flag, I put it here just as a reference for the 401 story.

Reply:
Ahh, I see. Can you please mention the ticket as well, since we have one, to avoid confusion to someone looking into the code?